dlme_airflow/utils/partition_url_builder.py

```diff
@@ -34,15 +34,7 @@ def urls(self):
     def _urls_from_provider(self):
         urls = [self.collection_url]
         expression = jsonpath_ng.parse(self.paging_config["urls"])
-        for match in expression.find(self.provider_data):
-            if [match.value for match in expression.find(self.provider_data)]:
-                page_urls = [
-                    match.value for match in expression.find(self.provider_data)
-                ][0]
-            for next in page_urls:
-                if next["url"]:
-                    urls.append(next["url"])
-
+        map(urls.append, [match.value for match in expression.find(self.provider_data)])
```

Comment:
I like how it was simplified. If you want you could also:

```python
urls.extend([match.value for match in expression.find(self.provider_data)])
```

Reply:
This is a good suggestion, thank you. I'll merge as is, but take a look at this in the follow up refactor.